axltempl/drupal.py

```diff
@@ -98,7 +100,7 @@ def main(
         name = name.split("/")
         name = name[1] if len(name) == 2 else name[0]
         util.writeInfo("Adding Lando support...")
-        lando.generateLandoFiles(name, docroot, cache)
+        lando.generateLandoFile(name, docroot, cache)
```

Comment:
@shinde-rahul, I'll still keep it plural as there are multiple files to be generated here, not just one.